analyzers/tests/ITs.JsonParser.Test/ITs.JsonParser.Test.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
We have this explicit everywhere. Should be good enough here too
```suggestion
```

Reply:
We have the same flag on the .NET8 test project.
It's fairly useful, especially in these testcases where you use `File`, `Directory`, `Path`, etc.

Reply:
That one is useless too. Remove it from Net8.
And add `System.IO` to `Usings` in this file below

Reply:
..btw you have `using System.IO` on few places that should go away :D